login.html

```diff
@@ -1,11 +1,44 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ko">
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css" />
+    <link rel="stylesheet" href="login.css">
     <title>Document</title>
 </head>
 <body>
-    로그인 페이지
+    <container>
+        <div id="logo">
+            <a href="index.html">
+                <img src="./image/판다얼굴.png">
+                <img src="./image/판다마켓.png">
+            </a>    
+        </div>
+        <form>
+            <lable>이메일</lable>
+            <input type="email" name="user-email" placeholder="이메일을 입력해주세요">
```

Comment:
❗️ 수정요청
특정 input이 어떤 값을 받는지 설명하는 text들을 label 이라고 합니다.
이러한 친구들은 일반적으로 마우스로 선택시 연결된 input으로 focus가 이동하는 기능이 있습니다~
이를 위해, 그리고 명확한 연결관계 표시하기 위해 label 태그를 사용해주시는 것이 좋습니다~

또한 label인데 lable로 오타가 난 것 같으니 수정해주세요~

```suggestion
            <label for='email'>이메일</label> // for 속성에 연결하고자하는 input id를 기입. 이렇게 하면 해당 태그 클릭시 연결된 input으로 포커스가 이동됩니다.
            <input type="email" id="email" name="user-email" placeholder="이메일을 입력해주세요">
```